roles/servicetelemetry/defaults/main.yml

```diff
@@ -44,6 +44,24 @@ servicetelemetry_defaults:
             storage_class: ""
             storage_selector: {}
             pvc_storage_request: 20Gi
+    logs:
+      loki:
+        enabled: false
+        replication_factor: 1
+        flavor: 1x.extra-small
+        storage:
+          object_storage_secret: ""
+          storage_class: ""
```

Comment:
This one looks fine once made consistent across the `ServiceTelemetry` manifest.